src/containers/Network/UpgradeStatus.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import axios from 'axios'
+import BarChartVersion from './BarChartVersion'
+import NetworkTabs from './NetworkTabs'
+import Streams from '../shared/components/Streams'
+import Hexagons from './Hexagons'
+import { localizeNumber } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+import Log from '../shared/log'
+
+export const UpgradeStatus = () => {
+  const [vList, setVList] = useState<any>([{}])
```

Comment:
Is there a better type than `any` to use here? (It's fine if there isn't :) )

Reply:
vList is retrieved from VHS and is quite complex so I put any for it.